templates/_helpers.yaml

```diff
@@ -70,12 +76,30 @@
         name: {{ $config.secretName }}
         key: {{ default "value" $config.secretKey }}
   {{- end }}
+{{- end }}
+
+{{- define "custom_airflow_environment_config" }}
+  # Dynamically created environment variables
+  {{- range $i, $config := .Values.env }}
+  - name: {{ $config.name }}
+    value: {{ $config.value | quote }}
+    {{- if eq $.Values.executor "KubernetesExecutor" }}
+  - name: AIRFLOW__KUBERNETES_ENVIRONMENT_VARIABLES__{{ $config.name }}
+    value: {{ $config.value | quote }}
+    {{- end }}
+  {{- end }}
   {{- if eq .Values.executor "KubernetesExecutor" }}
     {{- range $i, $config := .Values.secret }}
-  - name: AIRFLOW__KUBERNETES_SECRETS__{{ $config.envName }}
-    value: {{ printf "%s=%s" $config.secretName $config.secretKey }}
+    - name: AIRFLOW__KUBERNETES_SECRETS__{{ $config.envName }}
+      value: {{ printf "%s=%s" $config.secretName $config.secretKey }}
```

Comment:
@kushalmalani is this indentation right?

Reply:
Yeah, i was able to add those env vars to the pods and helm commands didnt throw any errors for that.